Description: Fix FTBFS on hurd-i386 which doesn't define PATH_MAX
Author: Bas Couwenberg <sebastic@xs4all.nl>
Last-Update: 2013-01-10
--- a/ogr/ogrsf_frmts/ili/iom/iom_p.h
+++ b/ogr/ogrsf_frmts/ili/iom/iom_p.h
@@ -69,6 +69,10 @@
    #define dbgnew   new
 #endif // _DEBUG
 
+#ifndef PATH_MAX
+#define PATH_MAX 4096
+#endif
+
 #ifdef _MSC_VER
 #define IOM_PATH_MAX _MAX_PATH
 #else
